### PR TITLE
formatTokenAmount() updates

### PR DESCRIPTION
**Pull request by bpierre**
This PR is to address @sohkai’s comments in https://github.com/aragon/aragon-ui/pull/728.

### CODE REVIEW

File: src/utils/math.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* global BigInt */
+
+import { divideRoundBigInt } from './math'
+
+describe('divideRoundBigInt()', () => {
+  test('should round the result', () => {
```
Comment (sohkai) [inline src/utils/math.test.js:6]: Might be worth adding another test just to smoke test the rounding works with a different base other than 10 (e.g. 5)

File: src/utils/format.test.js [hunk 2]
```diff
@@ -50,72 +50,111 @@ describe('formatNumber()', () => {
...
+    ).toEqual('2,839.00000001')
+    expect(
+      formatTokenAmount(BigInt('2839000000010000000000'), 18, { digits: 9 })
+    ).toEqual('2,839.00000001')
+    expect(
+      formatTokenAmount(BigInt('2839000000010000000000'), 18, { digits: 7 })
+    ).toEqual('2,839')
+  })
+
+  test('should throw when a negative number is used for decimals', () => {
```
Comment (sohkai) [inline src/utils/format.test.js:125]: Nitpick: what do you think about moving these "failure" expectations to the end?

File: src/utils/math.test.js (hunk 1 above)
Comment (bpierre) [reply src/utils/math.test.js:6]: Done https://github.com/aragon/aragon-ui/pull/760/commits/43ce5377977c2c288b3a8a1c3d7cb2ac38fc7df8

File: src/utils/format.test.js (hunk 2 above)
Comment (bpierre) [reply src/utils/format.test.js]: Done https://github.com/aragon/aragon-ui/pull/760/commits/ef9198496209ce5e45575bc0472b32dd9d8cc97c

**Pull request merged**